content/faqs/how-can-i-download-boundary-layers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How can I download boundary layers?
+tags: data
+category: getting-data
+---
+
+The following boundary layers can be downloaded from the Cal-Adapt API in GeoJSON format:
+
+
+[California Climate Change Fourth Assessment Regions](https://api.cal-adapt.org/api/ccc4aregions/)\
```

Comment:
I think the intent here is that URL be visible so that you can easily copy it into code, but maybe just better to make them jump one extra step. Thoughts?

Reply:
Could be worth verifying, but if that's the case it could be helpful to make the plain text URLs code blocks, like:

[California Climate Change Fourth Assessment Regions](https://api.cal-adapt.org/api/ccc4aregions/)
```
https://api.cal-adapt.org/api/ccc4aregions/
```

Reply:
But IMO it's still redundant as you can easily right click on the link and "copy URL"

Reply:
True. Ok, I will edit these out...

Reply:
I think we should call this good for now and maybe later come back to it.